clang/www/cxx_dr_status.html

```diff
@@ -3599,7 +3599,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/591.html">591</a></td>
     <td>CD4</td>
     <td>When a dependent base class is the current instantiation</td>
-    <td class="full" align="center">Clang 20</td>
+    <td class="unreleased" align="center">Clang 20</td>
```

Comment:
Issues that are resolved in Clang 20 should be marked as `"full"`, not `"unreleased"`. Other such cases below as well.

Reply:
This is caused by the fact that the script has hard-coded value of the latest release. I'll fix this